src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Migrations/OpenIdMigrations.cs

```diff
@@ -391,5 +405,120 @@ public async Task<int> UpdateFrom6Async()
 
             return 7;
         }
+
+        public async Task<int> UpdateFrom7Async()
+        {
+            // Create all index tables with the new collection value.  
+            SchemaBuilder.CreateMapIndexTable<OpenIdApplicationIndex>(table => table
+                .Column<string>(nameof(OpenIdApplicationIndex.ApplicationId), column => column.WithLength(48))
+                .Column<string>(nameof(OpenIdApplicationIndex.ClientId), column => column.Unique()),
+                collection: OpenIdApplicationCollection);
+
+            SchemaBuilder.AlterIndexTable<OpenIdApplicationIndex>(table => table
+                .CreateIndex("IDX_COL_OpenIdApp_DocumentId",
```

Comment:
I like what you did in all other places by adding `_COL` and just removing `Index`, so keeping the table name

Then when the table name is too long we can apply some acronym rules, i.e.  `WorkflowBlockingActivities => WFBlockingActivities`, or here but just as an example `OpenIdApplication => OIDApplication`

So here would be `IDX_COL_OpenIdApplication_DocumentId`, but not so important if too late ;)